BEST-PRACTICES.md

```diff
@@ -17,15 +17,24 @@ We recommend the following coding practices for high quality contributions:
 - Prefer self-explanatory code as much as possible, but provide helpful comments for complex expressions and code blocks.
 - Add unit tests for any new or changed functionality, and update any existing tests that are impacted by your changes.
   - SUSHI uses [Jest](https://jestjs.io/) as a testing framework.
-  - To run the full test suite, run `npm test`. Ensure all tests are passing.
+  - Ensure all tests are passing. Ensure that code coverage of the new code is complete.
+  - To run the full test suite, run `npm test`.
+  - To review the test coverage report, run `npm run coverage` after running the full test suite.
 - Follow the code style and conventions as enforced by the lint configuration and as evidenced by the existing code.
   - SUSHI uses [ESLint](https://eslint.org/) for code linting.
-  - To run the linter on all code, run `npm run lint`. Ensure there are no issues reported.
+  - Ensure there are no issues reported.
+  - To run the linter on all code, run `npm run lint`.
+  - To automatically fix as many issues as possible, run `npm run lint:fix`. This uses ESLint's [--fix](https://eslint.org/docs/latest/use/command-line-interface#fix-problems) option.
 - Follow the code formatting as enforced by the formatter configuration.
   - SUSHI uses [Prettier](https://prettier.io/) for code formating.
-  - To run the prettier on all code, run `npm run prettier`. Ensure there are no issues reported.
+  - Ensure there are no issues reported.
+  - To run Prettier on all code, run `npm run prettier`.
+  - To automatically rewrite files in order to resolve formatting issues, run `npm run prettier:fix`. This uses Prettier's [--write](https://prettier.io/docs/en/cli.html#--write) option.
 - Ensure any new dependencies do not contain any known security vulnerabilities
   - To check for known security vulnerabilities, we recommend using [npm-audit](https://docs.npmjs.com/cli/v10/commands/npm-audit). Run `npm audit` and ensure there are no new issues on your branch.
+- Ensure any new dependencies use the latest published version.
+  - If a new dependency is required but the latest published version cannot be used, add the dependency and reason for not updating to [DEPENDENCY-NOTES.md](DEPENDENCY-NOTES.md).
+  - To check the latest published version, check the versions of the package on [npm](https://www.npmjs.com/) or use [npm-outdated](https://docs.npmjs.com/cli/v10/commands/npm-outdated). Run `npm outdated` and check that the new dependency is not listed in the output.
```

Comment:
I like this addition, but maybe it should go _before_ the one about checking for known security vulnerabilities? Because if you check security vulnerabilities first and _then_ upgrade it to another version, you might be introducing another security vulnerability.

Reply:
Changed the order in [7c4b6fe](https://github.com/FHIR/sushi/pull/1474/commits/7c4b6fee2205e44567d370c0c1b861c3ba6a4746)